packages/ui/src/components/Button/Button.test.tsx

```diff
@@ -88,4 +89,24 @@ describe('#Button', () => {
 
     expect(screen.queryByRole('button')).toHaveClass(expected)
   })
+
+  it("shouldn't crash when wrapped with next/link", () => {
+    expect(() =>
+      render(
+        <Link href="https://supabase.com">
```

Comment:
This test was failing with before the change to the ref - doesn't seem related to passing a custom element with the `as` prop as originally theorized in https://github.com/supabase/supabase/pull/13540#issuecomment-1505152745